pkg/session/bootstrap.go

```diff
@@ -2892,6 +2897,19 @@ func upgradeToVer179(s Session, ver int64) {
 	doReentrantDDL(s, "ALTER TABLE mysql.global_variables MODIFY COLUMN `VARIABLE_VALUE` varchar(16383)")
 }
 
+func upgradeToVer180(s Session, ver int64) {
+	if ver >= version179 {
+		return
+	}
+
+	sql := fmt.Sprintf("INSERT HIGH_PRIORITY INTO %s.%s ("+
```

Comment:
Maybe `INSERT HIGH_PRIORITY IGNORE INTO mysql.GLOBAL_VARIABLES VALUES("tidb_txn_mode", "optimistic")` is enough.
No need to update when `VARIABLE_VALUE` is empty. /cc @you06

Reply:
fixed: https://github.com/pingcap/tidb/pull/48500/commits/4e263ed2a1bfcf8b78af13cf9805edad9386135f